src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +564,63 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
+  designReviewId   Int                  @id @default(autoincrement())
+  dateScheduled    DateTime
+  dateCreated      DateTime
+  userCreated      User                 @relation(fields: [userCreatedId], references: [userId], name: "designReviewCreator")
+  userCreatedId    Int
+  status           Design_Review_Status
+  teamType         Design_Review_Team
+  requiredMembers  User[]               @relation(name: "requiredAttendee")
+  optionalMembers  User[]               @relation(name: "optionalAttendee")
+  confirmedMembers User[]               @relation(name: "confirmedAttendee")
+  deniedMembers    User[]               @relation(name: "deniedAttendee")
+  location         String?
+  isOnline         Boolean
+  isInPerson       Boolean
+  zoomLink         String?
+  attendees        User[]               @relation(name: "userAttended")
+  dateDeleted      DateTime?
+  userDeleted      User?                @relation(fields: [userDeletedId], references: [userId], name: "designReviewDeleter")
+  userDeletedId    Int?
+  docTemplateLink  String?
+  wbsElementId     Int
+  wbsElement       WBS_Element          @relation(fields: [wbsElementId], references: [wbsElementId])
+}
+
+model Schedule_Settings {
+  drScheduleSettingsId Int          @id @default(autoincrement())
+  personalGmail        String
+  personalZoomLink     String
+  availability         Availability @relation(fields: [availabilityId], references: [availabilityId])
+  availabilityId       Int          @unique
+  User                 User         @relation(fields: [userId], references: [userId])
+  userId               Int          @unique
+}
+
+model Availability {
+  availabilityId        Int                @id @default(autoincrement())
+  mondayAvailability    DateTime[]
```

Comment:
ok, so instead of doing this, let's make availability an array of ints on schedule_settings, with each int corresponding to one 30-minute segment on the when2meet chart we're adding, starting indexing at 0. So like Monday at 9am would be 0, Tuesday at 9am would be 24, etc., we can discuss more at leads tn, but I think this will get us all the info we need, with a lot less potential for weird datetime issues